sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptionData.java

```diff
@@ -75,7 +95,7 @@ final class EncryptionData {
      *
      * @return encryption mode
      */
-    String getEncryptionMode() {
+    public String getEncryptionMode() {
```

Comment:
Given the class is package-private do the getter methods need to be public?

Reply:
Nope. My mistake.